sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/page_blob_client.hpp

```diff
@@ -208,6 +208,19 @@ namespace Azure { namespace Storage { namespace Blobs {
         const ResizePageBlobOptions& options = ResizePageBlobOptions(),
         const Azure::Core::Context& context = Azure::Core::Context()) const;
 
+    /**
+     * @brief Changes the sequence number for this page blob.
+     *
+     * @param action Specifies how the service should modify the blob's sequence number.
+     * @param options Optional parameters to execute this function.
+     * @param context Context for cancelling long running operations.
+     * @return An UpdateSequenceNumberResult describing the updated page blob.
+     */
+    Azure::Response<Models::UpdateSequenceNumberResult> UpdateSequenceNumber(
+        Models::SequenceNumberAction action,
+        const UpdatePageBlobSequenceNumberOptions& options = UpdatePageBlobSequenceNumberOptions(),
+        const Azure::Core::Context& context = Azure::Core::Context()) const;
+
```

Comment:
Hi @JeffreyRichter , can you please review the newly added API?

It was left out in GA release.